tests/doc_test/doc_needextend/index.rst

```diff
@@ -50,3 +50,10 @@ Need extend
 
 .. needextend:: extend_test_007
    :+links: extend_test_004, extend_test_005
+
+.. test:: needextend Example 8
+   :id: extend_test_008
+   :author: Foo_001
+
+.. needextend:: extend_test_008
+   :+author: Foo_002, Foo_003 
```

Comment:
The issue #350 is having its problems with "tag" option, which is a predefined field and stores the values in  a list.
I'm not sure if your solution works with tags.
Can you please update the test case and use "tag" as well?